ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1027,7 +1027,7 @@ class NoteTextComponent extends React.Component {
 			} else if (command.name === 'textCode') {
 				fn = this.commandTextCode;
 			} else if (command.name === 'insertTemplate') {
-				fn = () => { return this.commandTemplate(command.value); };
```

Comment:
No sorry you're right, it should be like you've done initially and it's also how it's done for focusElement below for instance. As long as the function is correctly bound (which it was in your code) it's fine. Sorry about that, it's a bit tricky to make sense of the diff sometime.

Reply:
No worries, I thought this might happen, it's just worthwhile to have the code in place.